files/usr/lib/systemd/system/jeos-firstboot.service

```diff
@@ -11,6 +11,7 @@ DefaultDependencies=no
 Conflicts=shutdown.target
 After=systemd-remount-fs.service
 Before=systemd-sysusers.service sysinit.target shutdown.target
+Wants=systemd-udev-settle.service
```

Comment:
added that a long time ago, still not sure if it's needed, the problem happened to be another thing, will remove for now